source/scpp/build.d

```diff
@@ -1,4 +1,10 @@
-#!/usr/bin/env rdmd
+#!/usr/bin/env dub
+/+
+ dub.json:
+ {
+     "name": "cpp_build"
```

Comment:
Question How does the part declared here affect?

Reply:
It does not, it's just a placeholder.